src/applications/mhv-medical-records/containers/VitalDetails.jsx

```diff
@@ -125,15 +132,23 @@ const VitalDetails = props => {
   useEffect(
     () => {
       if (records?.length) {
-        focusElement(document.querySelector('h1'));
+        // Update page title
```

Comment:
This comment seems unnecessary since it's the exact same wording as the function name below it.

Reply:
fixed